ui/components/MesheryPatterns.js

```diff
@@ -640,10 +639,14 @@ const mapStateToProps = (state) => {
 // @ts-ignore
 export default withStyles(styles)(connect(mapStateToProps, mapDispatchToProps)(withSnackbar(MesheryPatterns)));
 
+
+
 // --------------------------------------------------------------------------------------------------
 // -------------------------------------------- PATTERNS FORM ---------------------------------------
 // --------------------------------------------------------------------------------------------------
 
+
+
 function PatternForm({ pattern, onSubmit, show }) {
```

Comment:
@Abhishek-kumar09 @bariqhibat @leecalcote We are using this function `in` app`s as well but the main issue that arises is passing the values we need to assign values in every function accordingly to the file, for eg here we are using `patterns, pattern_file` so I think some of the function need to be replicated in both, please do correct my opinion if required.

Reply:
Yeah, in the current point of time we can change few things in the the components that will made this work as a single reusable component for both of the the pages. But if Application and pattern and filters has too much architecture difference, then need to separate out the things.